content/influxdb/v2.2/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,116 @@ menu:
+(https://github.com/influxdata/influxdb/commit/a7a5233): Advance queue scanner periodically instead of every remote write
+
+#### Performance?
+
+(https://github.com/influxdata/influxdb/commit/fea3037): Configure perf tests with yaml
+
+#### Metrics
+
+(https://github.com/influxdata/influxdb/commit/dece95d): Tsm compaction metrics via prometheus
+(https://github.com/influxdata/influxdb/commit/3a81166): Added metrics collection for replications
+(https://github.com/influxdata/influxdb/commit/6096ee2): Replications metrics include failure to enqueue
+(https://github.com/influxdata/influxdb/commit/0a74085): Point write requests have metrics
+(https://github.com/influxdata/influxdb/commit/a74e051): Disk size metrics per shard
+(https://github.com/influxdata/influxdb/commit/feb459c): Metrics for cache subsystem
+(https://github.com/influxdata/influxdb/commit/edb21ab): Metrics for wal subsystem
+(https://github.com/influxdata/influxdb/commit/b970e35): Remaining storage metrics from OSS engine
+
+#### Miscellaneous updates
+
+(https://github.com/influxdata/influxdb/commit/c51a0df): Error out when config file contains 1.x config values
+(https://github.com/influxdata/influxdb/commit/afb167a): `query-memory-bytes` zero-value is unlimited
+(https://github.com/influxdata/influxdb/commit/f78f9ed): Api/v2/config endpoint displays runtime configuration
+(https://github.com/influxdata/influxdb/commit/4f74049): Add downgrade target for 2.1
+
+### Bug fixes
+
+https://github.com/influxdata/influxdb/commit/84776d7): Manual task runs are scheduled asyncronously
+https://github.com/influxdata/influxdb/commit/5e6b0d5): Extend snapshot copy to filesystems that cannot link
+https://github.com/influxdata/influxdb/commit/88afa92): Detect misquoted tag values and return an error
+https://github.com/influxdata/influxdb/commit/2bace77): Unhandled errors returned by Sketch.Merge
+https://github.com/influxdata/influxdb/commit/fa9ba8e): Duplicated X-version and X-Build headers for /ping endpoint
+https://github.com/influxdata/influxdb/commit/8aa3a8f): Add causal error when meta.db is missing
+https://github.com/influxdata/influxdb/commit/799d349): Sync index file before close
+https://github.com/influxdata/influxdb/commit/5ce164f): Remove influx CLI output from CONTRIBUTING
+https://github.com/influxdata/influxdb/commit/e4e1633): Replications remote writes do not block server shutdown
+https://github.com/influxdata/influxdb/commit/39eeb3e): Fix race condition which could cause restore command to fail
+https://github.com/influxdata/influxdb/commit/e5cbd27): Advance replications queue after successful remote writes
+https://github.com/influxdata/influxdb/commit/4fd4bd0): Use copy when a rename spans volumes
+https://github.com/influxdata/influxdb/commit/11c0081): Disable use of jsonnet with `/api/v2/templates/apply`
+https://github.com/influxdata/influxdb/commit/0c30afd): Updating a check does not require an owner id
+
+### Other
+
```

Comment:
not sure how many of these others we want to mention, let's check with edge team